src/layouts/product-integration-layout/components/header/index.tsx

```diff
@@ -49,41 +59,83 @@ export default function Header({
 		.map((f: Flag) => f.slug)
 		.includes('builtin')
 
+	const tags = GetIntegrationTags(integration, true)
+
 	return (
-		<Card className={classNames(s.header, className)}>
-			<div className={s.upperCard}>
-				<div className={s.left}>
-					<h1>{integration.name}</h1>
-					<span>@{integration.organization.slug}</span>
-				</div>
-				<div className={s.right}>
-					{showVersions ? (
-						<DropdownDisclosure
-							className={s.versionDropdown}
-							color="secondary"
-							text={versionString(activeRelease.version, integration.versions)}
+		<Card elevation="base" className={s.headerWrap}>
+			<IconTileLogo
+				className={s.iconTile}
+				size="large"
+				productSlug={productSlug !== 'sentinel' ? productSlug : null}
+			/>
+			<div className={s.upper}>
+				{showVersions ? (
+					<DropdownDisclosure
+						className={s.versionSelector}
+						color="secondary"
+						text={versionString(activeRelease.version, integration.versions)}
+					>
+						{otherVersions.map((version: string) => {
+							return (
+								<DropdownDisclosureLinkItem
+									key={version}
+									href={getVersionChangedURL(version)}
+								>
+									{versionString(version, integration.versions)}
+								</DropdownDisclosureLinkItem>
+							)
+						})}
+					</DropdownDisclosure>
+				) : null}
+				<div className={s.titleDescriptionWrap}>
+					<div className={s.titleWrap}>
+						<TruncateMaxLines
+							maxLines={isMobile ? 2 : 1}
+							lineHeight="var(--token-typography-body-300-line-height)"
 						>
-							{otherVersions.map((version: string) => {
-								return (
-									<DropdownDisclosureLinkItem
-										key={version}
-										href={getVersionChangedURL(version)}
-									>
-										{versionString(version, integration.versions)}
-									</DropdownDisclosureLinkItem>
-								)
-							})}
-						</DropdownDisclosure>
-					) : null}
+							<Heading
+								size={300}
+								weight="bold"
+								level={1}
+								className={s.titleName}
+							>
+								{integration.name}
+							</Heading>
+						</TruncateMaxLines>
+						<Text size={100} weight="medium" className={s.titleOrganization}>
+							@{integration.organization.slug}
+						</Text>
+					</div>
+					<TruncateMaxLines
+						maxLines={2}
+						lineHeight="var(--token-typography-body-200-line-height)"
+					>
+						<Text size={200} className={s.description}>
+							{integration.description}
+						</Text>
+					</TruncateMaxLines>
 				</div>
 			</div>
-
-			<div className={s.lowerCard}>
-				<TagList size="medium" tags={GetIntegrationTags(integration, true)} />
+			<div className={s.lower}>
+				<ul className={s.tagList}>
+					{tags.map((tag: Tag) => (
+						<li key={tag.name} className={s.tag}>
+							<Tooltip label={tag.description}>
+								<Badge
```

Comment:
Used `Badge` here instead of the `TagList`. However, we should refactor `TagList` to render badges — the two different styles are already coded and handled by `Badge`.

Reply:
Good callout 💯 

🎟️ [Refactor TagList to render a list of 'badge' components](https://app.asana.com/0/1203792616809419/1203841143367699)